lib/github.ml

```diff
@@ -173,8 +173,9 @@ let run_with_auth ?(default_body = `Null) ~dry_run ~auth curl_t =
       args
     >>| fun () -> default_body
   else
-    OS.Cmd.must_exist (Cmd.v "curl") >>= fun _ ->
-    match Curly.(run ~args (Request.make ~url ~meth:`POST ())) with
+    OS.Cmd.must_exist (Cmd.v "curl") >>= fun curl ->
+    let exe = Cmd.to_string curl in
```

Comment:
If think `OS.Cmd.must_exist (Cmd.v "curl")` returns `"curl"`. Is that what you want? It seems that the `exe`-argument of `Curly.run` is `curl` anyways. If your intention was to use the explicit binary path (as in `/usr/bin/curl` or so), maybe `OS.Cmd.get_tool ` instead of `OS.Cmd.must_exist` and `Cmd.p curl` instead of `Cmd.to_string curl` would do.

Reply:
That's a good point. I was thinking that maybe if we passed the string to curly it would not have to guess too much things or check if it exists, but since it uses `"curl"` as default we don't need to do all of this. I'm not sure using the full path to the binary would be useful as it would need to be found in PATH anyway.
I guess it is a matter of whether we want to rely on Curly current default behavior or protect dune-release against changes in the future, and given the activity on Curly I think we are safe relying on current's behavior.

Reply:
Oh ok, I think I've misunderstood what you folks meant by "explicit binary" in the conversation about the README.